src/commonMain/kotlin/exchange.dydx.abacus/state/manager/StateManagerAdaptor.kt

```diff
@@ -1767,6 +1769,21 @@ open class StateManagerAdaptor(
         }
     }
 
+    fun adjustIsolatedMargin(
+        data: String?,
```

Comment:
what does it mean for data to be null here?

Reply:
It is possible to unset fields from the frontend by passing  `data: null`. AFAIK this is also used to initialize the `current` input. i.e. Setting the singleton from Trade -> Transfers -> Trade